lib/ansible/cli/galaxy.py

```diff
@@ -391,7 +391,7 @@ def execute_install(self):
             try:
                 installed = role.install()
             except AnsibleError as e:
-                display.warning(u"- %s was NOT installed successfully: %s " % (role.name, to_text(e)))
+                display.warning(u"- %s  and %s version was NOT installed successfully: %s " % (role.name, role.version, to_text(e)))
```

Comment:
remove 'and' and 'version' as the name + version should be good enough to identify and the 'and' is confusion as it seems you installed (or failed to ) 2 things.